### PR TITLE
Use Docker in Travis CI; publish container images to Quay

### PR DESCRIPTION
**Pull request by hectcastro**
Make use of the Travis CI support for Docker to execute the following tests within containers:

- Web
- Schema editor
- Application

In addition, setup image publishing to Quay on merges to `develop` that pass tests. Also, extract Windshaft container image definition to `quay.io/azavea/windshaft` and reference it.

See also:

- https://quay.io/repository/azavea/driver-app
- https://quay.io/repository/azavea/driver-editor
- https://quay.io/repository/azavea/driver-web
- https://travis-ci.org/WorldBank-Transport/DRIVER

### CODE REVIEW

**Comment by flibbertigibbet** [comment]
LGTM; maybe someone else would care to take a look, too.

File: app/driver/settings_travis.py [hunk 1]
```diff
@@ -15,11 +15,11 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.contrib.gis.db.backends.postgis',
-        'NAME': 'postgres',
-        'HOST': 'localhost',
+        'NAME': 'mub',
```
Comment (kshepard) [inline app/driver/settings_travis.py]: 'mub' -> 'prs' in several places.

**Comment by kshepard** [comment]
+1

File: app/driver/settings_travis.py (hunk 1 above)
Comment (hectcastro) [inline app/driver/settings_travis.py]: Ugh, my bad. Addressed in https://github.com/WorldBank-Transport/DRIVER/commit/47b6561597cbc6133b180f8a3ebe5f840a1c59e7.

**Pull request merged**